### PR TITLE
Update solidity-by-example.rst

### PR DESCRIPTION
**Pull request by varunagarwal315**
Might be trivial, but makes more sense to be able to directly return the name of the winner for the election. If the position of the winner on the proposal[] array is returned, then people still don't know the name of the person who won.

### CODE REVIEW

**Review by pirapira — CHANGES_REQUESTED** [review]
I think the new function can call `winningProposal()`, and then the code is much simpler.  Currently in this PR `winningProposal()` and `winnerName()` contain largely duplicate code.

**Comment by varunagarwal315** [comment]
Removed all the duplicate code as suggested and simply called the winningProposal() function. Added some comments to help explain the function to new users. Thanks pirapira for the suggestions

File: docs/solidity-by-example.rst [hunk 1]
```diff
@@ -170,6 +170,15 @@ of votes.
                 }
             }
         }
+        
+        //Calls winningProposal() function to get the index
+        //of the winner contained in the proposals array and then
+        //returns the name of the winner
```
Comment (pirapira) [inline docs/solidity-by-example.rst]: Just for uniformity, will you add a space between `//` and the following text?

**Comment by pirapira** [comment]
Thank you!  Let me compile it...

**Review by pirapira — APPROVED** [review]

**Comment by pirapira** [comment]
@chriseth is there a particular reason why we do not allow returning structs?  Or shall I create an issue about that?

**Comment by VoR0220** [comment]
@pirapira there is an issue for this, but basically the capability currently does not exist due to the ABI needing to be defined for structs.

**Comment by varunagarwal315** [comment]
@pirapira Are there any further changes to be made by me before this can be merged? Or is the change too trivial for a merge with dev branch.

Thanks

**Comment by pirapira** [comment]
@varunagarwal315 no, no, this was just a pause to hear people.

**Pull request merged**